plugin/ochttp/stats.go

```diff
@@ -60,8 +68,16 @@ var (
 	DefaultLatencyDistribution = view.Distribution(0, 1, 2, 3, 4, 5, 6, 8, 10, 13, 16, 20, 25, 30, 40, 50, 65, 80, 100, 130, 160, 200, 250, 300, 400, 500, 650, 800, 1000, 2000, 5000, 10000, 20000, 50000, 100000)
 )
 
-// Package ochttp provides some convenience views.
-// You need to subscribe to the views for data to actually be collected.
+// Convenience HTTP client views.
```

Comment:
Predefined HTTP client views.

There are not just convenience, we think these views are essential.

Reply:
There seems to be disagreement about this. I am going to remove the controversial parts to a separate PR.